kaggle-classification/keras_trainer/model.py

```diff
@@ -115,13 +115,11 @@ def predict(self, texts):
 
   def score_auc(self, data):
     predictions = self.predict(data['comment_text'])
-    scores = []
-    for idx, label in enumerate(LABELS):
-      labels = np.array(data[label])
-      score = metrics.roc_auc_score(labels, predictions[idx].flatten())
-      scores.append(score)
-      print('{} has AUC {}'.format(label, score))
-    print('Avg AUC {}'.format(np.mean(scores)))
+    labels = np.array(list(zip(*[data[label] for label in LABELS])))
```

Comment:
Whenever you do some fancy list comprehension magic, it's good to add a comment to explain to the uninitiated what the hell is going on :)